code-challenges.rb

```diff
@@ -10,6 +10,19 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def odd_or_even()
+    print "Input your number here."
+    num1=gets.chomp.to_i
+    print "Okay, let's see if #{num1} 's odd or even."
+        if num1.even? 
+            print "That's even."
+        else
+            print "That's not even."
+        end 
+    end
+
```

Comment:
Haha, I like the approach! You will still need to call your function in order to see the output.